README.md

```diff
@@ -1,152 +1,24 @@
-<a name="readme-top"></a>
+# README
 
```

Comment:
Please DO NOT change the README template. Please Only use the README template provided by Microverse 👍

Reply:
Thank you for the kind feedback. Issue has been fixed